projects/optic/src/commands/oas/captures/streams/sources/proxy.ts

```diff
@@ -41,7 +43,19 @@ export class ProxyInteractions {
       proxyPort?: number;
     }
   ): Promise<[ProxyInteractions, string, string]> {
-    let { host, protocol, origin } = new URL(targetHost);
+    let host: string;
+    let protocol: string;
+    let origin: string;
+    try {
+      ({ host, protocol, origin } = new URL(targetHost));
+    } catch (e) {
```

Comment:
should we make it an error to have a path? 

Since it's confusing to users and doesn't actually do anything?